cmd/wallet-js-sdk/src/universal/vc-wallet.js

```diff
@@ -390,6 +390,7 @@ export class UniversalWallet {
     },
     { from, timeout } = {}
   ) {
+    console.debug("about to call proposePresentation in vc-wallet for '"+this.user+"', myLabel: '"+myLabel+"', routerConnections: '"+routerConnections+"', reuseConnection:'"+reuseConnection+"', reuseAnyConnection:'"+reuseAnyConnection+"', connectionTimeout:'"+connectionTimeout+"'")
```

Comment:
instead of string concatenation in logs, try using commas.....`console.debug("about to call proposePresentation in vc-wallet for '", this.user,"', myLabel: '", myLabel, "'`.......javascript console prints are variadic

Reply:
removed all console.debug/log calls added in this change